doc/api/timers.md

```diff
@@ -162,6 +162,57 @@ added: v0.0.1
 
 Cancels a `Timeout` object created by [`setTimeout()`][].
 
+## Manual Timers
+
+It is possible, but not recommended, to make any object a timer. Under special
+circumstances this can be advantageous. However, in general practice, using
+this API is strongly discouraged. This is due to the fact that when using this API
+it is important to track timer enrollment state. A memory leak will occur if a
+timer is enrolled but never unenrolled.
+
+*Note*: This API is considered *unsafe*. Despite being public, it is subject
+to change at any time.
+
+Example:
+
+```js
+'use strict';
+
+const timers = require('timers');
+const atimer = {
+  _onTimeout: function() {
+    console.log('timeout');
+  }
+};
+
+timers.enroll(atimer, 1000); // make the `atimer` object a timer
+timers.active(atimer); // start the timer
+```
```

Comment:
This should include a clear example showing the timer being `unenroll()`'d per the warning above

Reply:
It does show it. Line  183, in the callback from the timer completion.